its/src/test/kotlin/org/sonarlint/intellij/its/tests/domain/OpenInIdeTests.kt

```diff
@@ -38,8 +38,8 @@ class OpenInIdeTests {
         fun createConnection(token: String) {
             with(remoteRobot) {
                 idea {
-                    dialog("Do you trust this SonarQube server?") {
-                        button("Connect to this SonarQube server").click()
+                    dialog("Trust This SonarQube Server?") {
```

Comment:
Was this change discussed, Trust this sound more like a command than a question :)

Reply:
IntelliJ guidelines [mention](https://jetbrains.github.io/ui/text/writing_short/#do-not-address-the-user) not to address the user, so we should avoid using "you" in sentences